api_mt/index.md

```diff
@@ -0,0 +1,137 @@
+# __HOW TO CONNECT POPULAR MTs TO CAT TOOLS__
```

Comment:
Let's use sentence case, we can always uppercase in CSS if we want

I'd avoid words like "popular", which are subjective.

Reply:
I will add this suggestion to our Machine Translate Style page.

Reply:
Ok. I have changed this part to make it compatible with your comment. It now looks as follows: How to connect MTs to CAT tools.